pkg/cloud/api/mutable_resource.go

```diff
@@ -180,6 +202,7 @@ func (u *mutableResource[GA, Alpha, Beta]) CheckSchema() error {
 	if err != nil {
 		return err
 	}
+	// TODOD(kl52752) Add validation that GA is a subset of Beta and Alpha.
```

Comment:
I'm assuming the PR for this is coming very soon?

Reply:
yes it's next in line